sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -295,6 +295,70 @@ export class ApplicationOfferingChangeRequestService {
     });
   }
 
+  /**
+   * Get the Application Offering Change Request Status by the application id.
+   * @param applicationId the application id.
+   * @param studentId the student id.
+   * @returns application offering change request status.
+   */
+  async getApplicationOfferingChangeRequestStatus(
+    applicationId: number,
+    studentId: number,
+  ): Promise<ApplicationOfferingChangeRequestStatus> {
+    const applicationOfferingChangeRequest =
+      await this.applicationOfferingChangeRequestRepo.findOne({
+        select: {
+          id: true,
+          applicationOfferingChangeRequestStatus: true,
+          createdAt: true,
+        },
+        relations: {
+          application: {
+            student: true,
+          },
+        },
+        where: {
+          application: { id: applicationId, student: { id: studentId } },
+        },
+        order: {
```

Comment:
do we need `orderby` if its `findOne`?

Reply:
@ann-aot I believe it is to guarantee the return of most recent application offering change.

Reply:
mm, Once an application offering request is declined or approved, the institution can again request another application offering request, right? so, that means, there can be multiple rows that needed to be displayed in the tracker and banners right? correct me if I am wrong.

Reply:
When there is a re-request, from my understanding it will replace the existing row of the previous request in the banner. There is only one row for each of the appeal and application offering change request in the banner. Please correct me if I am wrong.

Reply:
or do we only care about the latest request?